.vortex/CLAUDE.md

```
@@ -123,24 +123,24 @@ yarn lint
 yarn lint-fix
 ```
 
-**Template Tests** (`.vortex/tests/`):
+**Template Tests** (`.vortex/`):
 ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)